src/components/Proposal/ViewProposalContainer.tsx

```diff
@@ -54,6 +56,7 @@ class ViewProposalContainer extends React.Component<IProps, null> {
           <div className={css.singleProposal}>
             <ProposalContainer proposalId={proposal.proposalId}/>
           </div>
+          <Route exact path="/dao/:daoAvatarAddress/proposal/:proposalId" component={ViewSharingButtonsContainer} />
```

Comment:
My understanding is that the proposal was for the buttons to show up on the individual proposal pages.

Reply:
Well, I tried it, and I've got some incorrect assumptions somewhere. I believed that it would need the route in order to keep from showing on pages other than the individual proposal pages.